spec/create-markdown.py

```diff
@@ -163,7 +163,13 @@ def createNote(r, g):
         md_string += func.comment + "\n\n"
         if func.seeAlso:
             md_string += "**See also**<br>"
-            md_string += func.seeAlso.replace("http://www.w3.org/2000/10/swap/","").replace("#",":") + "\n\n"
+            url = func.seeAlso
+            parsed_url = urllib.parse.urlparse(url)
+            fragment = parsed_url.fragment
+            last_component = fragment.split('/')[-1] 
```

Comment:
(wouldn't the last component be proceeded by "#"?)

Reply:
Nop, it's OK. Please remember that we need only last word.